app/views/pages/about.html.erb

```diff
@@ -5,6 +5,8 @@
 <main role="main" id="content">
   <%= render partial: 'layouts/feedback_survey_banner' if flash[:beta_message] == 'hide'%>
 
+  <div class="text">
+
   <h1 class="heading-large">
```

Comment:
can you fix the indentation inside that div?

Reply:
I purposefully didn't add the indentation so that the diff was readable.

There is unnecessary duplication between each page that I'd like to fix by refactoring out some shared layouts but I've saved that for a later date.

Reply:
True it's harder to read, but it's a problem with git and it shouldn't dictate how we write our code. Proper indentation has saved us more than once. You can also reindent in a different commit, or just ignore whitespace changes in git diff

Reply:
Point taken. Fixed with 84d72557f5052da8c8c2c742535a4f3ad8b7a263